docs/samples/add_satellite_node.yml

```diff
@@ -56,17 +56,17 @@
       register: expansion_status
 
     - name: Check if satellite node expansion job is completed...The node addition is still ongoing.
-      async_status:
+      ansible.builtin.async_status:
```

Comment:
Hi, thanks for the PR.
Is this part necessary? It seems it's not related to FQCN?